docs/integrations/ai-engines/lightwood.mdx

```diff
@@ -52,7 +52,7 @@ The values vary between 0 and 1, where 1 indicates a perfect predictor, based on
 You can check accuracy values for models using the [`DESCRIBE`](/sql/api/describe) statement.
 </Note>
 
-## Tuning the Lightwood ML Engine Features
+## Parameters' Tuning
```

Comment:
Nitpick: change to `Tuning the Lightwood ML Engine`? I think it sounds a bit more aligned with the content that follows.